packages/launchpad/src/checklist-item/index.tsx

```diff
@@ -28,6 +29,13 @@ const ChecklistItem = ( { task, isPrimaryAction }: { task: Task; isPrimaryAction
 	// when clicking on the task list items.
 	const buttonHref = task.useCalypsoPath && task.calypso_path ? task.calypso_path : undefined;
 
+	// The Button component does not accept the `disabled` and `href` props together.
+	// This code will only add href property if the disabled variable is false.
+	const buttonProps = {
+		...( disabled ? { disabled } : {} ),
+		...( disabled ? {} : { href: buttonHref } ),
+	};
```

Comment:
Based on https://github.com/WordPress/gutenberg/blob/a99f3264efa305b018aeb34785706b023f1acf2c/packages/components/src/button/types.ts#L136, it looks like we can include `disabled: false` for the Anchor/`a` case, and `disabled: false|true` for the Button case, so it may be worth simplifying the handling for `disabled`. 
```suggestion
	// The Button component does not accept the `disabled` and `href` props together.
	// This code will only add href property if the disabled variable is false.
	const buttonProps = {
		disabled,
		...( disabled ? {} : { href: buttonHref } ),
	};
```